std_msgs/Header header
string group_name
string state_name
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status